jasmine/tests/BUILD.bazel

```diff
@@ -16,15 +16,8 @@ jasmine_test(
 )
 
 jasmine_test(
-    name = "config_no_chdir",
+    name = "config_chdir_package_name",
     data = ["config_file.test.js"],
     config = "config_file.js",
-    chdir = None,
```

Comment:
this test helps us paths.relativize() works with "." as well. by default `chdir =  package_name()` to prevent jasmine from scanning `wksp/external` under runfiles. 

Is there any reason this shouldn't be the behavior?

Reply:
https://github.com/aspect-build/rules_jasmine/pull/7/files#r938349218 includes `chdir = package_name()` portion

Reply:
I see the source of confusion now.

Generally we want to leave chdir with a None default in our rulesets so we are consistent. Its already strange to chdir to the output tree so we should at least be consistent with that default across rules_js rulesets.

Reply:
Not scanning in folders that you don't want to scan in would then be up to the user to set the correct spec filter.

Reply:
Hmm. sounds good. then perhaps we should rule out external via `!external/**` in the generated config file. or we could not do that at all.

Reply:
I do like the path.relativize() for the config. We should bring that improvement to jest_test.

Reply:
`--nolegacy_external_runfiles` gets rid of the `wksp/external` folder;

generally I don't think `**/*.spec.js` from the root of a workspace is a good idea.